test/sagemaker_tests/pytorch/training/integration/sagemaker/test_smppy.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2018-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from __future__ import absolute_import
```

Comment:
You import `absolute_import` but still use relative imports. why?

Reply:
i used the existing scripts in the repo as a template, in particular the shared imports at the top. i am not certain on all aspects of their style but i thought it would be more confusing if only my own test had different imports.